src/components/liveSession/services/liveSession.service.js

```diff
@@ -249,24 +245,43 @@
             };
 
             this._updateLesson = (liveSessionData) => {
+                function updateSingleLesson(lesson) {
+                    // update lesson startTime, endTime and status
+                    lesson.startTime = lesson.startTime ? lesson.startTime : liveSessionData.startTime ? liveSessionData.startTime: null;
```

Comment:
startTime & endTime => move to lessonNotes obj.
please call these: actualStartTime & acturalEndTime